src/test/java/org/mifos/integrationtest/common/Utils.java

```diff
@@ -7,7 +7,8 @@
 public class Utils {
 
     public static final String TENANT_PARAM_NAME = "Platform-TenantId";
-    public static final String DEFAULT_TENANT = "ibank-india";
+    public static final String REQUEST_TYPE_PARAM_NAME = "requestType";
+    public static final String DEFAULT_TENANT = "gorilla";
```

Comment:
can we get these from configs, it would enable us to change it without a code change

Reply:
Unable to pick up application.yml value and assign it here.

Reply:
todo: not here. this is a Util file, in Utils we usually have static fields,  the dynamic fields like tenant name we can fetch and pass to here as arguments.